src/test/java/hudson/plugins/git/GitStatusCrumbExclusionTest.java

```diff
@@ -32,6 +34,16 @@
     private HttpServletResponse resp;
     private FilterChain chain;
 
+    @BeforeClass
+    public static void setProps() {
+        System.setProperty("hudson.security.csrf.CrumbFilter.UNPROCESSED_PATHINFO", "true");
```

Comment:
Isn't it safer to get the old value and restore it?

Reply:
I updated the code. Feel free to commit directly to the PR.

Reply:
When I checked out this pull request and modified pom.xml to allow it to compile with Jenkins 2.204.6, I get a null pointer exception in the `unsetProps` call.

```
java.lang.NullPointerException
        at java.util.Hashtable.put(Hashtable.java:460)
        at java.util.Properties.setProperty(Properties.java:166)
        at java.lang.System.setProperty(System.java:798)
        at hudson.plugins.git.GitStatusCrumbExclusionTest.unsetProps(GitStatusCrumbExclusionTest.java:47)
```

I think this PR and https://github.com/jenkins-infra/jenkins.io/pull/3002 highlight that it is time for the git plugin and the git client plugin to require a Jenkins version that is newer than 2.138.4.

I think that we should consider incrementing the major version of the git plugin and the git client plugin and have them depend on Jenkins 2.190.3.  I think that would allow us to avoid several dependency related issues that are caused by the overly broad range of Jenkins versions supported by the git plugin and the git client plugin.

@fcojfernandez and @rsandell do you have concerns if we make git plugin require Jenkins 2.190.3 instead of 2.138.4? I'll also start a discussion in the [git-plugin gitter channel](https://gitter.im/jenkinsci/git-plugin)

Reply:
@MarkEWaite Good catch ;) corrected